Demo/Components/Button/MapSettings/MapSettingsButtonDemoView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+import FinniversKit
+
+public class MapSettingsButtonDemoView: UIView {
+
+    private lazy var mapSettingsButton: MapSettingsButton = {
```

Comment:
Could be just `private lazy var mapSettingsButton = MapSettingsButton(withAutoLayout: true)`